Sample Data/Custom/NCProtectUAL_CL.json

```diff
@@ -0,0 +1,244 @@
+[
+	{
+		"TimeGenerated [UTC]": "3/21/2021, 10:38:57.794 PM",
+		"SHA512Hash_g": "",
```

Comment:
As per the column name suffix (_g) its value should be guid. If value is empty for guid type then give it in this format "00000000-0000-0000-0000-000000000000".

Please update accordingly in all the field.

Thanks,
Aditi Maheshwari

Reply:
We have update the sample data file and made the required changes in the file. Please find the sample data file below and  you can use this file for reference.
[NCProtectUAL.zip](https://github.com/Azure/Azure-Sentinel/files/6286312/NCProtectUAL.zip)

Thanks,
Aditi Maheshwari

Reply:
Hi @v-admahe Thanks for your update and sorry for delay.